docs/lisk-tools/embedded-wallet-providers.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Embedded wallet providers
+description: Summary of embedded wallets provides that support the Lisk network.
+keywords:
+  [
+    Lisk embedded wallets,
+    embedded wallets,
+    embedded wallet,
+    embedded wallet providers,
+    web3 wallet,
+    wallet development,
+    Panna,
+    Privy, 
+    ThirdWeb,
+    Xellar
+  ]
+---
+
+# Embedded wallet providers
+
+This page summarizes providers of embedded wallets that support the Lisk network.
+
+Embedded wallets providers support developers to integrate web3 wallets directly into an application, rather than requiring a separate browser extension or app. They allow users to take action without leaving the host application.
```

Comment:
```suggestion
Embedded wallet providers support developers to integrate web3 wallets directly into an application, rather than requiring a separate browser extension or app. They allow users to take action without leaving the host application.
```